Tests/Blazorise.E2E.Tests/Blazorise.E2E.Tests.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
Prefer to not use ImplicitUsings

Reply:
Why not? It's a simple test project. Reduces boilerplate.

Reply:
I.E very clean files:
![image](https://user-images.githubusercontent.com/22283161/229097972-15b112b1-d7cf-47ae-a15b-8d3dddb2e41f.png)

Straight to the point, testing.

Reply:
Not a fan. TBH.

Reply:
Can you concede in this project, since it is not a "releasable" version? :) 
Or would you still prefer that I change?

Reply:
🤐OK. You can leave it 😅